pkg/cmd/service/service.go

```diff
@@ -169,7 +169,7 @@ func DefaultConfig() *Config {
 			Enabled:            true,
 			Addr:               ":8080",
 			TLS:                TLSConfig{Enabled: false},
-			UpstreamTimeout:    3 * time.Second,
+			UpstreamTimeout:    5 * time.Second,
```

Comment:
I have seen this take more than three seconds testing locally on serverless Aurora on a region far away.